docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -0,0 +1,213 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:snip6: https://github.com/ericnordelo/SNIPs/blob/feat/standard-account/SNIPS/snip-6.md[SNIP-6]
+
+= Account
+
+Reference of interfaces, presets, and utilities related to Account contracts.
+
+== Core
+
+[.contract]
+[[ISRC6]]
+=== `++ISRC6++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/account/interface.cairo#L12[{github-icon},role=heading-link]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::interface::ISRC6;
+```
+
+Interface of the ISRC6 standard as defined in the {snip6}.
+
+
+[.contract-index]
+.Functions
+--
+* xref:#ISRC6-\\__execute__[`++__execute__(calls)++`]
+* xref:#ISRC6-\\__validate__[`++__validate__(calls)++`]
+* xref:#ISRC6-is_valid_signature[`++is_valid_signature(hash, signature)++`]
+--
+
+[.contract-item]
+[[ISRC6-__execute__]]
+==== `[.contract-item-name]#++__execute__++#++(calls: Array<Call>) → Array<Span<felt252>>++` [.item-kind]#external#
+
+Executes the list of calls as a transaction after validation.
+
+Returns the list of each call output.
+
+[.contract-item]
+[[ISRC6-__validate__]]
+==== `[.contract-item-name]#++__validate__++#++(calls: Array<Call>) → felt252++` [.item-kind]#external#
+
+Validates a transaction before execution.
+
+Returns the short string `'VALID'` if valid, or any other value otherwise.
+
+[.contract-item]
+[[ISRC6-is_valid_signature]]
+==== `[.contract-item-name]#++is_valid_signature++#++(hash: felt252, signature: Array<felt252>) → felt252++` [.item-kind]#external#
+
+Validates whether a signature is valid or not for the given message hash.
+
+Returns the short string `'VALID'` if valid, or any other value otherwise.
+
+[.contract]
+[[Account]]
+=== `++Account++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/account/account.cairo#L27[{github-icon},role=heading-link]
+
+:OwnerAdded: xref:Account-OwnerAdded[OwnerAdded]
+:OwnerRemoved: xref:Account-OwnerRemoved[OwnerRemoved]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::Account;
+```
+Base Account contract implementation.
```

Comment:
```suggestion
Account contract implementation extending xref:ISRC6[`ISRC6`].
```
to keep stressing on the idea that ISRC6 are valid and sufficient accounts. otherwise it might look like this is the most basic account.